### PR TITLE
ui: Fix win_hide distination grid

### PR DESCRIPTION
**Pull request by tksrsk**
When create tab, win_hide sent to new tab and new tab's previous tab.
So, if created tab that not next tab (eg. $tabnew cmd),
send win_hide to not current tab, and fixed this.

### CODE REVIEW

File: src/nvim/window.c [hunk 1]
```diff
@@ -3787,6 +3787,7 @@ void free_tabpage(tabpage_T *tp)
 /// @return Was the new tabpage created successfully? FAIL or OK.
 int win_new_tabpage(int after, char_u *filename)
 {
+  tabpage_T   *old_curtab = curtab;
   tabpage_T   *tp = curtab;
```
Comment (bfredl) [inline src/nvim/window.c:3791]: lets move this variable inside the else block line 3814 and use `old_curtab` for the other usages, to be consequent.

**Comment by bfredl** [comment]
LGTM otherwise.

File: src/nvim/window.c (hunk 1 above)
Comment (tk-shirasaka) [reply src/nvim/window.c]: Thank you for review, fixed it!

**Comment by bfredl** [comment]
merged, thanks!

**Pull request merged**